onnxscript/function_libs/torch_aten/graph_building.py

```diff
@@ -444,3 +480,7 @@ def to_model_proto(
         onnx.checker.check_model(onnx_model, full_check=True)
         print("[Success] ONNX model exported")
         return onnx_model
+
+    def apply(self, graph_pass: Callable, *args, **kwargs):
```

Comment:
Future usage?

Reply:
Yes. I updated to fx side to reflect this